packages/app/src/components/Comments/Comments.js

```diff
@@ -6,35 +6,31 @@ import { FormatQuote } from '@material-ui/icons'
 
 const Comments = ({ placeId }) => {
   const [restaurantReviews, setReviews] = useState()
-  const [showReviews, setShowReviews] = useState()
 
   useEffect(() => {
     const getReviews = async () => {
       if (placeId) {
         try {
           const reviews = await commentService.getCommentsForRestaurant(placeId)
           setReviews(reviews)
-          setShowReviews(reviews.rating !== undefined)
         } catch (error) {
-          setShowReviews(false)
+          console.log(error)
```

Comment:
clogging is always a bit iffy, but I'm not sure how to do proper error handling here